pallets/funding/src/functions.rs

```diff
@@ -71,24 +74,30 @@ impl<T: Config> Pallet<T> {
 	/// # Next step
 	/// The issuer will call an extrinsic to start the evaluation round of the project.
 	/// [`do_start_evaluation`](Self::do_start_evaluation) will be executed.
-	pub fn do_create(issuer: &AccountIdOf<T>, initial_metadata: ProjectMetadataOf<T>) -> DispatchResult {
+	pub fn do_create(
+		issuer: &AccountIdOf<T>,
+		initial_metadata: ProjectMetadataOf<T>,
+		_did: DID,
```

Comment:
I expected there to be more features to include that need credentials, so just in case passing them to use later on. 
If we already agree credential use to be as it is, we can remove them yes

Reply:
I think its mostly as is now. We can always pass them later if we actually need them. Its inner functions so nothing for extrinsics will change, so its a simpel runtime upgrade if we want to add additional logic. Lets keep is simpel for now